account_multicurrency_revaluation/demo/account_demo.xml

```diff
@@ -1,66 +1,105 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="res_partner_reval" model="res.partner">
+        <field name="name">Revaluation Company</field>
+        <field eval="1" name="customer"/>
+        <field eval="1" name="supplier"/>
+        <field name="is_company">1</field>
+        <field eval="1" name="active"/>
+        <field name="street">90 Streets Avenue</field>
+        <field model="res.country" name="country_id" search="[('code','ilike','ch')]"/>
```

Comment:
Would it be possible to use ``base.ch`` instead?
```
<field name="country_id" ref="base.ch"/>
```
?

Reply:
Yep. I think it is a better solution.